doc/ref_model/chapters/chapter07.md

```diff
@@ -507,3 +519,191 @@ Table 7-2 shows security capabilities
     tools
 -   Resiliency tests run (such as hardware failures or power failure
     tests)
+
+<a name="7.11"></a>
+## 7.11 Consolidated Security Requirements
+
+Please note that **Virtual Resource Manager (VRM)** is used to collectively represent VIM, K8s, etc.
```

Comment:
I think the definition of VRM should be clear, as in what it includes. Using ```etc.``` is open-ended.

Reply:
@msidana Given the move away from NFVI (will edit), VRM will be changed to "Cloud Infrastructure Manager (CIM)"

Reply:
@msidana Changes made